### DISCUSSION TITLE
Types declarations for SNS payloads from CloudWatch alarms

### DISCUSSION THREAD

**Discussion by jpage-godaddy** [aws/aws-sdk-js-v3#3775] [Q&A]
I'm looking for reference material or better yet TypeScript types that describe the SNS payloads for CloudWatch alarm actions, but all I seem to be able to find is samples. Are these payloads described _anywhere_? After a lot of searching I'm not sure if it's not documented or if I'm missing the right keywords to find this information.

**Comment by CanRau** [comment]
I'm looking for the same just for SES http payloads like bounces etc

looks like I'm going to turn the examples into types myself, surprising they're not officially maintained

https://stackoverflow.com/questions/47643871/aws-lambda-sns-trigger-event-type
